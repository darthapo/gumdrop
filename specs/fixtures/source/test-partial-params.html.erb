<%= render 'user-params', name:"Matt" %>