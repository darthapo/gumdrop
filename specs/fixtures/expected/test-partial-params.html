<body><div>User Matt</div></body>